jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcAdapterProperties.java

```diff
@@ -64,6 +66,12 @@ private static String getProperty(final Map<String, String> properties, final St
         }
     }
 
+    public static List<String> getIgnoreErrorList(final Map<String, String> properties) {
+        String ignoreErrors = getProperty(properties, PROP_IGNORE_ERROR_LIST, "");
```

Comment:
Maintainability and interface stability: Overload `getProperty()` with another method that does not require an empty string as parameter.

Reply:
done